### ISSUE TITLE
[BUG] Progress keeps spinning in detached mode

### ISSUE THREAD

**Issue by solarless** [docker/compose#11639]
### Description

When running `docker compose up` the network and volume creation progress ui works correctly, i.e. it spins and then (in most cases almost immediately) gets marked as created with green check mark **and stops**.

But when running `docker compose up -d` this progress isn't working correctly.
Network or volume creation progress starts spinning but never stops, although it gets marked as created but without green check mark.

### Steps To Reproduce

Basically `docker compose up -d`

### Compose Version

```Text
Docker Compose version v2.25.0
```

### Docker Environment

```Text
Client: Docker Engine - Community
 Version:    25.0.5
 Context:    default
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc.)
    Version:  v0.13.1
    Path:     /usr/libexec/docker/cli-plugins/docker-buildx
  compose: Docker Compose (Docker Inc.)
    Version:  v2.25.0
    Path:     /usr/libexec/docker/cli-plugins/docker-compose

Server:
 Containers: 0
  Running: 0
  Paused: 0
  Stopped: 0
 Images: 18
 Server Version: 25.0.5
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Using metacopy: false
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Cgroup Version: 1
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: ae07eda36dd25f8a1b98dfbf587313b99c0190bb
 runc version: v1.1.12-0-g51d5e94
 init version: de40ad0
 Security Options:
  seccomp
   Profile: builtin
 Kernel Version: 5.15.146.1-microsoft-standard-WSL2
 Operating System: Ubuntu 22.04.4 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 6
 Total Memory: 7.717GiB
 Name: DESKTOP-IC2FCSK
 ID: 024f069d-5a40-4b86-b861-869735c1476a
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false

WARNING: No blkio throttle.read_bps_device support
WARNING: No blkio throttle.write_bps_device support
WARNING: No blkio throttle.read_iops_device support
WARNING: No blkio throttle.write_iops_device support
```

### Anything else?

Running just `docker compose up`:
```
$ docker compose up
[+] Running 7/7
 ✔ Network azerothcore-wotlk_default     Created                                                                   0.3s
 ✔ Network azerothcore-wotlk_ac-network  Created                                                                   0.3s
 ✔ Container ac-database                 Created                                                                   3.2s
 ✔ Container ac-client-data-init         Created                                                                   3.2s
 ✔ Container ac-db-import                Created                                                                   1.7s
 ✔ Container ac-authserver               Created                                                                   2.4s
 ✔ Container ac-worldserver              Created                                                                   2.5s
```

Running in detached mode:
```
$ docker compose up -d
[+] Running 5/7
 ⠼ Network azerothcore-wotlk_ac-network  Created                                                                  28.5s
 ⠙ Network azerothcore-wotlk_default     Created                                                                  28.2s
 ✔ Container ac-database                 Healthy                                                                  19.8s
 ✔ Container ac-client-data-init         Exited                                                                    8.9s
 ✔ Container ac-db-import                Exited                                                                   19.6s
 ✔ Container ac-authserver               Started                                                                  23.1s
 ✔ Container ac-worldserver              Started                                                                  23.4s
```

As you can see the networks `azerothcore-wotlk_ac-network` and `azerothcore-wotlk_default` have been created but progress has not been stopped

I am using Ubuntu on WSL, but I have the same issue on pure Ubuntu (not WSL)

**Comment by felixfontein** [comment]
I also noticed that while toying around with #11656.

**Comment by arcanisgk** [comment]
(Workaround)
The same thing happened to me and it was necessary to separate the construction and the initialization:

```
docker-compose ... create --build
docker-compose ...  up -d
```

It's been a while since I saw someone messing around with pirate World of Warcraft servers...

**Comment by ndeloof** [comment]
AFAICT this issue has been fixed by https://github.com/docker/compose/pull/11725

**Comment by solarless** [comment]
Now it works correctly from 2.26.1!

As far as I understand, #11725 wasn't included in the 2.26.1 release so I don't know what fixed this issue
